test.runsettings

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RunSettings>
+   <MSTest>
+      <!-- Specify timeout in milliseconds. A valid value should be greater than 0 -->
+      <TestTimeout>300000</TestTimeout>
```

Comment:
Check - do the long running tests require > 5mins to complete?

Reply:
Current timeout is set to 6mins (Long running tests - DeviceClient_TokenIsRefreshed_Ok_Mqtt, DeviceClient_DeviceDisabled_Gives_ConnectionStatus_DeviceDisabled_AMQP take a little over 5mins to complete.)